tms/clickup/admin.py

```diff
@@ -1,3 +1,6 @@
 from django.contrib import admin
+from django.contrib.auth.models import User
 
-# Register your models here.
+
+admin.site.unregister(User)
```

Comment:
what is unregister ?
why did you right unregister and then register ?

Reply:
i don't unregistered user that time error show is already user register because of cookie cutter bydulate create user app and this app use auth user so i write unregister user

Reply:
No need to register user here it is automatically done by django